pom.xml

```diff
@@ -835,15 +835,11 @@
               <artifactId>checkstyle</artifactId>
               <version>6.19</version>
             </dependency>
-            <dependency>
-              <groupId>org.apache.beam</groupId>
-              <artifactId>beam-sdks-java-build-tools</artifactId>
-              <version>${project.version}</version>
-            </dependency>
           </dependencies>
           <configuration>
-            <configLocation>beam/checkstyle.xml</configLocation>
-            <suppressionsLocation>beam/suppressions.xml</suppressionsLocation>
+            <!-- beam.rootdir set by scjijava:set-rootdir -->
+            <configLocation>${beam.rootdir}/sdks/java/build-tools/src/main/resources/beam/checkstyle.xml</configLocation>
```

Comment:
Without the scijava plugin, this path would be evaluated relative to each directory, and it would be impossible to express a path that works. So the effect of this is to get what @dhalperi  asked me for, which was to get checkstyle looking at the top-level POM. It also enables us to remove the complexity of the separate module.